lib/net/ssh/proxy/jump.rb

```diff
@@ -10,7 +11,7 @@ module Proxy
       #
       #   require 'net/ssh/proxy/jump'
       #
-      #   proxy = Net::SSH::Proxy::Jump.new('user@proxy')
+      #   proxy = Net::SSH::Proxy::Jump.new('user@proxy', options: ["StrictHostKeyChecking=no"])
```

Comment:
Another option here would be making this more generic like:

```ruby
proxy = Net::SSH::Proxy::Jump.new('user@proxy', flags: ["-o StrictHostKeyChecking=no"])
```

Then users could pass in any SSH flags which would be more flexible.